strongbox-security/strongbox-authentication-providers/strongbox-default-authentication-provider/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.carlspring.strongbox</groupId>
+        <artifactId>strongbox-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>../../../strongbox-parent</relativePath>
+    </parent>
+
+    <artifactId>strongbox-default-authentication-provider</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Strongbox: Default Authentication Provider</name>
```

Comment:
I know it may look a bit odd, but in such aggregators (or sub-aggregators) for similar things, in the `<name/>` tag, I normally define things like this:

`foo/pom.xml`:
```
    <name>Strongbox: Authentication Provider [Foo]</name>
```

`bar/pom.xml`:
```
    <name>Strongbox: Authentication Provider [Bar]</name>
```

So for LDAP and so on, it will be:
`foo/pom.xml`:
```
    <name>Strongbox: Authentication Provider [LDAP]</name>
```

The formatting and ordering of the words looks odd. I admit. However, when you go over a build log with tons of modules, it improves the readability.

Would you mind making this cosmetic change (here and in the default implementation) please? Otherwise, it looks good! :)